dotnet/src/SemanticKernel.Abstractions/Orchestration/FunctionResult.cs

```diff
@@ -1,7 +1,11 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
 using System;
+using System.Collections.Concurrent;
 using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Reflection;
 
 namespace Microsoft.SemanticKernel.Orchestration;
```

Comment:
We should move this to Microsoft.SemanticKernel. That's where Kernel lives, KernelFunction lives, and every invocation on those gives you back one of these.

Reply:
Done